CONTRIBUTING.md

```diff
@@ -27,5 +23,17 @@
 * Interactive gh-pages ([http://anguscroll.com/just](anguscroll.com/just))
   * We'll take care of this after we land your PR and publish the npm module
 
+# README template
+* Thanks to [Max Synnott](github.com/maxsynnott) local and global READMEs are now autogenerated from a template.
+* Add a new section to [md-variables.com](https://github.com/angus-c/just/blob/master/md-variables.json) for your package:
+  *  root key: the `name` field from your `package.json` (e.g. `just-comapct`)
+  *  `packageName`: same as root key
+  *  `dir`: the directory you wrote your package in (e.g. `array-compact`)
+  *  `description`: a one sentence description of what the utility does
```

Comment:
@angus-c Just a note that the description also supports multiple lines. If an array of strings is provided, each element will be displayed on it's own line